images/modifiers/nvidia.yq

```diff
@@ -0,0 +1 @@
+.init += ["NVIDIA_TAG"]
```

Comment:
Nice. This is a whole lot easier to understand with the `image/modifiers/` structure.